authentication.md

```diff
@@ -91,6 +91,7 @@ This will return the authentication status, E.G.:
 {
   "okay" : true,
   "authenticated" : true,
+  "allowOnBehalfOf" : false,
```

Comment:
sorry to haven't got the comment immediately where you have anticipated such change. I want to suggest to use the new authorization endpoint to check for the possibility to use the loginAs feature. You can implement a loginAs feature that support the Site object to know if the current user is entitled or not to access such feature.